src/components/Projects/Router.tsx

```diff
@@ -0,0 +1,46 @@
+import { Route, Switch } from 'react-router-dom';
```

Comment:
or maybe just index.tsx?

Reply:
There's already an index file with the projects list view. I suppose I can make the router `index.tsx` and that file the `ProjectsView.tsx`... Or maybe this file should go somewhere else? Feels kinda weird having a router in `src/components` in the first place

Reply:
We have a few of those routers under components. Maybe move them to a `src/routes`. Fine to leave as-is for now.

Reply:
OK I will move it in a future PR when we add another route to Projects